packages/sw-precaching/src/lib/models/precache-entries/base-precache-entry.js

```diff
@@ -1,14 +1,14 @@
-import {cacheBustParamName} from '../constants';
+import {cacheBustParamName} from '../../constants';
 
 class RevisionedCacheEntry {
-  constructor({revisionID, revision, request, cacheBust}) {
-    this.revisionID = revisionID;
+  constructor({entryID, revision, request, cacheBust}) {
+    this.entryID = entryID;
```

Comment:
It could be helpful to use the `assert` library to validate the parameters if there are any that are required and/or need to be of a specific type.